src/js/store.js

```diff
@@ -0,0 +1,18 @@
+import { createStore, compose, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import promiseMiddleware from 'redux-promise-middleware';
+import rootReducer from './reducers/index';
```

Comment:
The `index.js` file is looked for automatically so this could be `import rootReducer from './reducers'`